en/flight_controller/modalai_fc_v1.md

```diff
@@ -0,0 +1,111 @@
+
+More information about the firmware can be found [here](https://docs.modalai.com/flight-core-firmware/).
+
+## PX4 Bootloader
+
+This board is supported in the PX4 bootloader starting with v5.0 [here](https://github.com/PX4/Bootloader).
+
+More information about the bootloader can be found [here](https://docs.modalai.com/flight-core-bootloader/).
+
+## QGroundControl Support
+
+This board is slated to be supported starting in QGroundControl 3.5.7 (or the release after 3.5.6).
+
+## Specification
+
+| Feature          | Details |
+|:--------         |:------- |
+| Weight           | 6 g     |
+| MCU              | 216MHz, 32-bit ARM M7 [STM32F765II](https://www.st.com/en/microcontrollers-microprocessors/stm32f765ii.html) |
+| Memory           | 256Kb FRAM   |
+|                  | 2Mbit Flash  |
+|                  | 512Kbit SRAM |
+| Firmware         | [PX4](https://github.com/PX4/Firmware/tree/master/boards/modalai/fc-v1) |
+| IMUs             | [ICM-20602](https://www.invensense.com/products/motion-tracking/6-axis/icm-20602/) (SPI1) |
+|                  | ICM-42688 (SPI2) |
+|                  | [BMI088](https://www.bosch-sensortec.com/bst/products/all_products/bmi088_1) (SPI6) |
+| Barometer        | [BMP388](https://www.bosch-sensortec.com/bst/products/all_products/bmp388) (I2C4) |
+| Secure Element   | [A71CH](https://www.nxp.com/products/security-and-authentication/authentication/plug-and-trust-the-fast-easy-way-to-deploy-secure-iot-connections:A71CH) (I2C4) |
+| microSD Card     | [Information on supported cards](https://dev.px4.io/v1.9.0/en/log/logging.html#sd-cards) |
+| Inputs           | GPS/Mag |
+|                  | Spektrum |
+|                  | Telemetry |
+|                  | CAN bus   |
+|                  | PPM    |
+| Outputs          | 6 LEDs (2xRGB) |
+|                  | 8 PWM Channels |
+| Extra Interfaces | 3 serial ports |
+|                  | I2C |
+|                  | GPIO |
+
+## Dimensions
+
+![FlightCoreV1Dimensions](../../assets/flight_controller/modalai/fc_v1/dimensions.png)
+
+## Availability
+
+- Flight Core Complete Kit is available [here](https://shop.modalai.com/products/flight-core)
+- Flight Core Board Only is available [here](https://shop.modalai.com/products/flight-core-pcb-only)
+- Flight Core assembled with VOXL and cameras is available at [here](https://shop.modalai.com/products/voxl-flight-deck-r1)
+
+## Quick Start
+
+### Orientation
+
+Below shows the orientation when using the recommended version listed above.
+
+- **NOTE** - the version 1.10 release of PX4 requires 'ROTATION_YAW_180' whereas the release maintained by ModalAI and in PX4 master does not
```

Comment:
What I think you are saying here is that for master you'd set `ROTATION_NONE` but for PX4 v 1.10 you'd set `ROTATION_YAW_180`. Is that right?

Reply:
Correct... basically I'd call what made it's way into 1.10 for orientation on our end was a 'bug' in some ways, and I've since updated it, such that `ROTATION_NONE` is used moving forward for the given orientation in the picture (this can be seen here, where the cameras are the front of the vehicle: https://shop.modalai.com/products/voxl-flight-deck-r1)

Reply:
Thanks. OK, I'll merge then slightly tidy this text :-). Thanks